<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Showcustomer</class>
 <widget class="QWidget" name="Showcustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>361</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Debt_L">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>217</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Debt Amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Username_LE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>47</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
	padding-left: 5px;
}

QLineEdit:focus {
    border-color: rgb(52, 81, 98);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name_LE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>117</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
	padding-left: 5px;
}

QLineEdit:focus {
    border-color: rgb(52, 81, 98);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Username_L">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>27</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Debt_LE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>237</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
	padding-left: 5px;
}

QLineEdit:focus {
    border-color: rgb(52, 81, 98);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Customerid_L">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>22</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(6, 106, 110);
border: none;
border-radius: 12px;
font: 700 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>#</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Lastname_L">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>97</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Lastname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Address_LE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>297</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
	padding-left: 5px;
}

QLineEdit:focus {
    border-color: rgb(52, 81, 98);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Name_L">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>97</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="Credit_L">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>217</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Credit Amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Credit_LE">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>237</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
	padding-left: 5px;
}

QLineEdit:focus {
    border-color: rgb(52, 81, 98);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Phonenumber_L">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>157</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Phonenumber</string>
    </property>
   </widget>
   <widget class="QLabel" name="Address_L">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>277</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lastname_LE">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>117</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
	padding-left: 5px;
}

QLineEdit:focus {
    border-color: rgb(52, 81, 98);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Edit_PB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>342</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 137, 64);
    border: none;
    border-radius: 7px;
	color: rgb(31, 48, 58);
	font: 600 11pt &quot;Segoe UI&quot;;
}
QPushButton:hover {
    background-color: rgb(255, 215, 147);
}
QPushButton::pressed {
    background-color: rgb(255, 85, 0);
}</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_PB">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(6, 106, 110);
    border: none;
    border-radius: 5px;
	font: 600 11pt &quot;Segoe UI&quot;;
}
QPushButton:hover {
    background-color: rgb(7, 132, 136);
}
QPushButton::pressed {
    background-color: rgb(5, 97, 100);
}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Phonenumber_LE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>177</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
	padding-left: 5px;
}

QLineEdit:focus {
    border-color: rgb(52, 81, 98);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_PB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 137, 64);
    border: none;
    border-radius: 7px;
	color: rgb(31, 48, 58);
	font: 600 11pt &quot;Segoe UI&quot;;
}
QPushButton:hover {
    background-color: rgb(255, 215, 147);
}
QPushButton::pressed {
    background-color: rgb(255, 85, 0);
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password_L">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_LE">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
	padding-left: 5px;
}

QLineEdit:focus {
    border-color: rgb(52, 81, 98);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
